drfc_manager/models/storage_operations.py

```
@@ -1,45 +1,44 @@
         storage_client.upload_hyperparameters(model_data.hyperparameters)
         storage_client.upload_model_metadata(model_data.metadata)
-        
+
         if model_data.reward_code:
             storage_client.upload_reward_function(model_data.reward_code)
         else:
             storage_client.upload_reward_function(model_data.reward_function)
     except Exception as e:
-        raise StorageError(f"Error uploading model data: {e}")
\ No newline at end of file
+        raise StorageError(f"Error uploading model data: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)